src/TSTransformer/util/jsx/getJsxIndex.ts

```diff
@@ -0,0 +1,18 @@
+import luau from "@roblox-ts/luau-ast";
+import assert from "assert";
+import { TransformState } from "TSTransformer/classes/TransformState";
+
+function getJsxIndex(factory: string) {
+	const [namespace, index] = factory.split(".");
+	return luau.property(luau.id(namespace), index);
```

Comment:
What if the factory is set to `a.b.Fragment`? Is that allowed by TS, and if so, should roblox-ts support that?

Reply:
Added support for this